shared/api/resource.go

```diff
@@ -551,6 +554,19 @@ type ResourcesNetworkCardSRIOV struct {
 	VFs []ResourcesNetworkCard `json:"vfs" yaml:"vfs"`
 }
 
+// ResourceNetworkCardVDPA represents the VDPA configuration of the network card
+//
+// swagger:model
+//
+// API extension: ovn_nic_acceleration_vdpa.
+type ResourcesNetworkCardVDPA struct {
```

Comment:
@gabrielmougard unless I'm mistaken, this PR doesn't yet populate this struct?

Reply:
no indeed

Reply:
I'll fix that